content/en/docs/setup/platform-setup/kubesphere/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: KubeSphere Container Platform
+description: Instructions to setup a KubeSphere Container Platform for Istio.
+weight: 18
```

Comment:
@sdake I also noticed it's ascii-betically sorted, since I see the weight of `Kubernetes Gardener` is 19. What's the weight value of KubeSphere should be?

Reply:
Also see that there is another PR for `kind` which is changing some weights in this area. It would be good to coordinate.

Reply:
> Also see that there is another PR for `kind` which is changing some weights in this area. It would be good to coordinate.

@ericvn 

Yes, I see [kind's PR](https://github.com/istio/istio.io/pull/6416/files) has set its weight to 17 and change IBM Cloud's weight from `18` to `16`, so may I leave `KubeSphere`'s weight as 18?  How to coordinate it?

In this way, the sequence will be 
- IBM Cloud
- Kind
- KubeSphere
- Kubernetes Gardener

Reply:
> Also see that there is another PR for `kind` which is changing some weights in this area. It would be good to coordinate.

Hi @ericvn, 

I remove the weight changes from IBM Cloud for coordination, please help to review this PR.

Thanks

Reply:
One item to note is that after checking in a change, the `deploy/netlify` test will run again. Others can use this to see what the site would look like with your changes. It's better for you just to run the `make serve` and then use a browser locally to see the output.

Reply:
> One item to note is that after checking in a change, the `deploy/netlify` test will run again. Others can use this to see what the site would look like with your changes. It's better for you just to run the `make serve` and then use a browser locally to see the output.

Got it, will preview site locally to verify it, Thanks for reminding it.